### ISSUE TITLE
Feature: Add support for Vultr Builder

### ISSUE THREAD

**Issue by koalalorenzo** [hashicorp/packer#8041]
It would be nice to have [Vultr](https://vultr.com) support integrated without being forced to manually build & add the Vultr plugin.

Source: https://github.com/vultr/packer-builder-vultr

**Comment by rickard-von-essen** [comment]
Just a note, you don't have to compile it since there is a ready made release to download.

**Comment by ddymko** [comment]
Hey all 👋 

Just like @rickard-von-essen each release has the binaries readily available on https://github.com/vultr/packer-builder-vultr

@azr I just wanted to follow up and see if there is anything required from me to get the vultr plugin merged into packer? I followed the structure and setup of other providers and tried to match it closely. If there is something missing/required please let me know!

**Comment by SwampDragons** [comment]
I also commented on the PR, but I'll repost here so people don't need to bounce through the issue tracker to know what's going on: 

> 
> Internally, we've been having a conversation about whether it makes sense to keep merging third party builders that we (HashiCorp) don't have the engineering resources to officially support -- I think it's been causing confusion in the community to have the officially-supported and community-supported plugins in the same place, and it makes the issue tracker a bit cluttered.  
> 
> I'm working on a roadmap that will hopefully come up with a solution that makes managing plugins a bit more seamless so that users don't have to worry about compiling, etc... but I think for now at least we're not going to be merging new builders until that path forward has been figured out. 
> 
> I'm happy to leave this open while we're deliberating on that, but it'll be a while and my suspicion at this point is that we're going to end up saying "keep it separate".

**Comment by SwampDragons** [comment]
Ultimately, this is one of those situations where we've been angsting internally about Packer growing far beyond our capacity to maintain it for _a while_ and we're finally at a point where we have decided we need to stop merging things for now while we figure out what a better-organized plugin system looks like.

**Comment by ddymko** [comment]
@SwampDragons No worries thanks for the response! If the decision is made to no longer merge in providers into packer then feel free to close up the PR. We will still continue supporting an official plugin from Vultr either way.

**Comment by SwampDragons** [comment]
Thanks so much for your understanding!

**Comment by rickard-von-essen** [comment]
Maybe add it to this list:
https://packer.io/community-tools.html#third-party-plugins

**Comment by SwampDragons** [comment]
It's already on there, isn't it?

**Comment by ddymko** [comment]
Yup, it looks like that page is already updated to point to the Vultr maintained plugin
